cpp/benchmarks/flow_simulation_ode_seir.cpp

```diff
@@ -32,61 +32,92 @@ namespace mio
 namespace benchmark
 {
 
-using FlowModel = ::mio::oseir::Model;
+using FlowModel = ::mio::oseir::Model<ScalarType>;
 
 using namespace oseir;
 
 // For comparison benchmarks, an old model version that does not provide computation of flows has been reimplemented here.
 // For more details see the original implementation in:
 // https://github.com/SciCompMod/memilio/blob/13555a6b23177d2d4633c393903461a27ce5762b/cpp/models/ode_seir/model.h
-class FlowlessModel : public CompartmentalModel<InfectionState, Populations<InfectionState>, Parameters>
+class FlowlessModel
+    : public CompartmentalModel<ScalarType, oseir::InfectionState,
+                                Populations<ScalarType, AgeGroup, oseir::InfectionState>, oseir::Parameters<ScalarType>>
 {
-    using Base = CompartmentalModel<InfectionState, mio::Populations<InfectionState>, Parameters>;
+    using InfectionState = oseir::InfectionState;
+    using Parameters     = oseir::Parameters<>;
+    using Base           = CompartmentalModel<ScalarType, oseir::InfectionState,
+                                    mio::Populations<ScalarType, AgeGroup, oseir::InfectionState>,
+                                    oseir::Parameters<ScalarType>>;
 
 public:
-    FlowlessModel()
-        : Base(Populations({InfectionState::Count}, 0.), ParameterSet())
+    FlowlessModel(const Populations& pop, const ParameterSet& params)
+        : Base(pop, params)
+    {
+    }
```

Comment:
This is unused, remove it.

Reply:
it is indirectly used by the second constructor

Reply:
Right, I overlooked that. But you can just use "Base" directly in the second constructor instead of going through this one.

I am not saying this is a bad idea in general, but since this model is only used for benchmarking, I'd like to keep it as simple as possible.